oak_containers_hello_world_untrusted_app/src/lib.rs

```diff
@@ -13,12 +13,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use oak_containers_launcher::Launcher;
-
 mod app_client;
 
-/// The local address that will be forwarded by the VMM to the guest on port 8080.
-const UNTRUSTED_APP_ADDRESS: &str = "http://127.0.0.1:8088";
+use oak_containers_launcher::Launcher;
+use std::net::Ipv4Addr;
+
+/// The local address that will be forwarded by the VMM to the guest's IP adress.
+const PROXY_ADDRESS: Ipv4Addr = Ipv4Addr::LOCALHOST;
```

Comment:
Should this constant be defined in the `oak_containers_launcher` crate? In fact, should the launcher have a method to return an already opened socket on the correct port and address, so we can avoid this entirely?

Reply:
Good idea. For now I just moved it to the launcher but do not return an open socket, only a socket address for now.